fenix/app/src/main/res/values/strings.xml

```diff
@@ -2349,11 +2403,13 @@
     <string name="download_languages_header_learn_more_preference">Learn more</string>
     <!-- The subhead of the download language preference screen will appear above the pivot language. -->
     <string name="download_languages_available_languages_preference">Available languages</string>
-    <!-- A text in parenthesis that will appear next to the pivot language -->
-    <string name="download_languages_default_system_language_require_preference">required</string>
+    <!-- A text in parenthesis that will appear next to the pivot language. -->
+    <string name="download_languages_default_system_language_require_preference">(required)</string>
+    <!-- A text in parenthesis that will appear next to a general language. The first parameter is the language file size, for example, "(3.91 KB)". -->
+    <string name="download_languages_general_language_preference">(%1$s)</string>
```

Comment:
This should be something like `%1$s (%2$s)` where the first parameter is the Language, and the second is the file size

Reply:
The same.